Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine@sha256:9a839e63dad54c3a6d1834e29692c8492d93f90c59c978c1ed79109ea4fb9a54
+FROM alpine:3.12.0@sha256:185518070891758909c9f839cf4ca393ee977ac378609f700f60a771a2dfe321
```

Comment:
Where’s that SHA from? Looks like the latest is this:

https://hub.docker.com/layers/alpine/library/alpine/3.12.0/images/sha256-a15790640a6690aa1730c38cf0a440e2aa44aaca9b0e8931a9f2b0d7cc90fd65?context=explore

Reply:
when I run `docker pull alpine:3.12.0` I get `185518070891758909c9f839cf4ca393ee977ac378609f700f60a771a2dfe321`

Reply:
I also get the same: 
```docker pull alpine:3.12.0
3.12.0: Pulling from library/alpine
Digest: sha256:185518070891758909c9f839cf4ca393ee977ac378609f700f60a771a2dfe321
Status: Downloaded newer image for alpine:3.12.0
docker.io/library/alpine:3.12.0```

Reply:
🤷‍♂️